src/common.ml

```diff
@@ -20,17 +20,16 @@ type token =
 
 (* Constants *)
 type const =
-      AtomConst of string    (* atoms    *)
-    | IntConst of int        (* integers *)
-    | FloatConst of float    (* floats   *)
-    | StringConst of string  (* strings  *)
```

Comment:
Let's keep `IntConst`, `FloatConst`, and `StringConst` for now. You don't need to handle them in your evaluator yet, but I don't want to dumb down my lexer if I've already done the work.